### ISSUE TITLE
Wrong license reported for ts-xlsx

### ISSUE THREAD

**Issue by dereklin** [davglass/license-checker#99]
ts-xlsx's license is GNU AFFERO GENERAL PUBLIC LICENSE: https://github.com/DxCx/ts-xlsx/blob/master/LICENSE

But license-checker reports MIT:

```
c:\Temp\foo>license-checker
├─ @types/xlsx@0.0.34
│  ├─ licenses: MIT
│  ├─ repository: https://wwwhub.com/DefinitelyTyped/DefinitelyTyped
│  └─ licenseFile: c:\Temp\foo\node_modules\@types\xlsx\LICENSE
├─ foo@1.0.0
│  └─ licenses: ISC
└─ ts-xlsx@0.0.11
   ├─ licenses: MIT
   ├─ repository: https://github.com/DxCx/ts-xlsx
   ├─ publisher: Hagai Cohen
   └─ licenseFile: c:\Temp\foo\node_modules\ts-xlsx\LICENSE

```

**Comment by davglass** [comment]
This is a tricky one, their `package.json` reports `MIT`:
http://registry.npmjs.com/ts-xlsx/latest

`license-checker` will go by what's in the `package.json` and if it's not defined there it will then try to guess it from the LICENSE file in the repo.

**Comment by dereklin** [comment]
Thanks for pointing that out.  I'll open an issue with ts-xlsx on this.
